docs/resources/stripe_meter.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: "stripe"
+page_title: "Stripe: stripe_meter"
+description: |-
+  The Stripe Meter can be created, modified and configured by this resource.
+---
+
+# stripe_meter
+
+With this resource, you can create a billing meter - [Stripe API billing meter documentation](https://docs.stripe.com/api/billing/meter).
+
+A billing meter is a resource that allows you to track usage of a particular event. For example, you might create a billing meter to track the number of API calls made by a particular user. You can then attach the billing meter to a price and attach the price to a subscription to charge the user for the number of API calls they make.
+
+Related guide: [Usage based billing](https://docs.stripe.com/billing/subscriptions/usage-based)
+
```

Comment:
Would be beneficial to inform users that delete isn't supported thus only deactivation happens. Please have a look at other resources how this is done.

Reply:
Could you point me to an example? I check price and product and couldn't find anything like what you are describing.

Reply:
https://registry.terraform.io/providers/lukasaron/stripe/latest/docs/resources/stripe_price

Reply:
https://registry.terraform.io/providers/lukasaron/stripe/latest/docs/resources/stripe_product

Reply:
https://registry.terraform.io/providers/lukasaron/stripe/latest/docs/resources/stripe_shipping_rate

Reply:
done